packages/react-components/src/organisms/UserProfileBiography.tsx

```diff
@@ -13,32 +15,42 @@ const stretchOnMobile = css({
 });
 
 type UserProfileBiographyProps = {
-  readonly biography: string;
+  readonly biography?: string;
   readonly biosketch?: string;
 };
 const UserProfileBiography: React.FC<UserProfileBiographyProps> = ({
   biography,
   biosketch,
-}) => (
-  <Card>
-    <div
-      css={{
-        display: 'flex',
-        flexDirection: 'column',
-      }}
-    >
-      <Headline2 styleAsHeading={3}>Biography</Headline2>
-      <Paragraph accent="lead">{biography}</Paragraph>
-      {biosketch && (
-        <div css={stretchOnMobile}>
-          <Link buttonStyle href={biosketch}>
-            {docsIcon}
-            View Biosketch
-          </Link>
-        </div>
-      )}
-    </div>
-  </Card>
-);
+}) => {
+  const { isOwnProfile } = useContext(UserProfileContext);
+  return biography || isOwnProfile ? (
```

Comment:
What if no bio but biosketch is there?

Reply:
I just maintained what was there before?